summarizer_server/server.py

```diff
@@ -40,6 +50,70 @@ def summarize():
     return json.dumps(top_sentences)
 
 
+@app.route("/api/feedback/submit", methods=["POST"])
+def submit_feedback():
+    if not request.is_json:
+        abort(400)
+
+    request_payload = request.get_json()
+    url = request_payload.get("url", "")
+    score = request_payload.get("score", "")
+
+    if url == "" or score == "":
+        abort(400)
+
+    email = request_payload.get("email", "")
+    gaia = request_payload.get("gaia", "")
+
+    # check if user wishes to be anonymous
+    user = userservice.get_anonymous()
+    if email != "" and gaia != "":
+        user = userservice.get(email)
+
+    # create user if it does not yet exist
+    if user is None:
+        user = userservice.create(email, gaia)
```

Comment:
I wonder if you can combine this into a single call, like `userservice.get_or_create(email, gaia)`. This seems to be the current behaviour of `userservice.create`, so what would you think about renaming that and using it here to replace the 2 calls?

Reply:
Sounds good, done!